meta-leda-distro/wic/qemuarm64.wks

```diff
@@ -27,7 +27,7 @@
 
 bootloader --ptable gpt --configfile="qemuarm64.cfg"
 
-part --source bootimg-partition --ondisk vda --fstype=vfat --label boot --active --align 4 --size 16 --sourceparams="loader=u-boot" --use-uuid
+part --source bootimg-partition --ondisk vda --fstype=vfat --label boot --active --align 4096 --size 100
```

Comment:
If it's really label in uppercase, we should add a comment here an also use "--label BOOT" just in case?

Reply:
Also I see that "--use-uuid" has been removed, that may also be causing the problem with "LABEL=BOOT" not working:
https://patchwork.yoctoproject.org/project/oe-core/patch/20220422094448.27342-1-henning.schild@siemens.com/

Reply:
Added back